starter-code/MongooseMovies/bin/seeds.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/celebrity', {useMongoClient: true}); // not sure if we need the second part
+const Celebrity = require('../models/celebrity');
+
+const celebrities = [
+  {
+    name: "Famous Frank",
+    occupation: "Actor",
+    catchphrase: "I'm so famous, I'm Frank"
+  },
+  {
+    name: "Celebrity Charlie",
+    occupation: "Actress",
+    catchphrase: "Catch me if you can"
+  },
+  {
+    name: "Annoying Annie",
+    occupation: "Model",
+    catchphrase: "Don't hate me because I'm beautiful"
+  },
+];
+
+Celebrity.create(celebrities, (err, celebArray) =>{ // what does .create do?
```

Comment:
Model.create is a method of the mongoose model class. It's a way to manually create instances of that model in the database. We are using it here to create a initial db so we can develop on it, but in our actual API we are using model_instance.save() instead.